tools/docker/docker-compose.yml

```diff
@@ -8,11 +8,20 @@ services:
       dockerfile: tools/docker/Dockerfile
     ports:
       - "8080:8080"
-    volumes:
-      - sqlite_db:/opt/app-root/src/sqlite
     environment:
       - AE_HOST=0.0.0.0
-      - AE_DATABASE_URL=sqlite:///./sqlite/test.db
-      - AE_ASYNC_DATABASE_URL=sqlite+aiosqlite:///./sqlite/test.db
+      - AE_DATABASE_URL=postgres+asyncpg://postgres:secret@postgres/ansible-events
+    depends_on:
+      - postgres
+
+  postgres:
+    image: "docker.io/library/postgres:14"
```

Comment:
Is there any special reason to use v14? this version is not available yet on RHEL AFAIK, if it's possible to use 13, that would make the things easier to our mates of AAP.

Reply:
Thanks, sure, we can use v13. There is no specific reason to use v14.